community/sway/usr/share/sway/scripts/sunset.sh

```diff
@@ -23,6 +23,10 @@ start() {
         latitude=${latitude:-$(echo "$GEO_CONTENT" | jq '.latitude // empty')}
         latitude=${latitude:-$fallback_latitude}
 
+        # Remove double quotes
```

Comment:
Good find, but I am relatively sure you can achieve the same easier by calling `jq` with `-r` for raw output.